packages/mobile/src/identity/privacy.ts

```diff
@@ -190,3 +209,42 @@ export function* getUserSelfPhoneHash() {
 
   return PhoneNumberUtils.getPhoneHash(e164Number, salt)
 }
+
+function* navigateToQuotaPurchaseScreen() {
+  try {
+    yield new Promise((resolve, reject) => {
+      navigate(Screens.PhoneNumberLookupQuota, {
+        onBuy: resolve,
+        onSkip: reject,
+      })
+    })
+
+    const ownAddress: string = yield select(currentAccountSelector)
+    const txId = generateStandbyTransactionId(ownAddress)
+    yield put(
+      transferStableToken({
+        recipientAddress: ownAddress, // send payment to yourself
+        amount: '0.01', // one penny
+        comment: 'Lookup Quota Purchase',
+        txId,
+      })
+    )
+
+    const result = yield call(waitForTransactionWithId, txId)
```

Comment:
Nit: I would find this easier to read if the variable name was more descriptive than `result`

Reply:
Will do